src/openforms/forms/api/serializers/form.py

```diff
@@ -245,16 +329,79 @@ def get_fields(self):
+            fields["registration_backend"].allow_blank = True
+            fields["registration_backend_options"] = fields[
+                "registration_backends"
+            ].child.get_fields()["options"]
+            fields["registration_backend_options"].required = False
+            fields["registration_backend_options"].allow_null = True
+
         return fields
 
     def validate(self, attrs):
         super().validate(attrs)
-        self.validate_backend_options(
-            attrs,
-            "registration_backend",
-            "registration_backend_options",
-            registration_register,
-        )
+        if "registration_backend" in attrs and "registration_backends" in attrs:
+            backend = attrs.pop("registration_backend")
+            options = attrs.pop("registration_backend_options")
+            match attrs["registration_backends"]:
+                case []:
+                    # ambiguous "new" call
+                    raise serializers.ValidationError(
+                        {
+                            "registration_backend": ErrorDetail(
+                                _(
+                                    "registration_backend is deprecated, please use just registration_backends",
+                                ),
+                                code="invalid",
+                            ),
+                            "registration_backend_options": ErrorDetail(
+                                _(
+                                    "registration_backend is deprecated, please use just registration_backends",
+                                ),
+                                code="invalid",
+                            ),
+                        }
+                    )
+                case [
+                    dict(backend=first_backend, options=first_options, name=_, key=_),
+                    *_,
+                ]:
+                    # could be our own backwards compatible export reimported
+                    # checking for ambiguity
+                    if backend != first_backend or options != first_options:
+                        raise serializers.ValidationError(
+                            {
+                                "registration_backend": ErrorDetail(
+                                    _(
+                                        "registration_backend is deprecated, please use just registration_backends",
+                                    ),
+                                    code="invalid",
+                                ),
+                                "registration_backend_options": ErrorDetail(
+                                    _(
+                                        "registration_backend is deprecated, please use just registration_backends",
+                                    ),
+                                    code="invalid",
+                                ),
+                            }
+                        )
```

Comment:
Can we simplify our life and make this mutual exclusive? Just always say use one or the other.

Old exports should only have `registration_backend` and `registration_backend_options`. New exports should only have `registration_backends` (so make sure that the old fields are dropped in new exports?). We don't support importing newer formats into older versions anyway, backwards compatibility is already hard enough without handling forwards compatibility :stuck_out_tongue: